optashift-employee-rostering-gwtui/src/main/java/org/optaplanner/openshift/employeerostering/gwtui/client/resources/i18n/OptaShiftUIConstants.java

```diff
@@ -5,7 +5,7 @@
 public interface OptaShiftUIConstants {
 
     @TranslationKey(
-            defaultValue = "The server returned an empty result which is impossible.\nMaybe the server is using the wrong WildFly version. Try using WildFly 10.1.0.Final.")
+            defaultValue = "The server returned an empty result which is impossible.\nMaybe the server is using the wrong WildFly version.")
```

Comment:
Should mention a working WildFly Version (if not here somewhere else where it is easily accessible)

Reply:
Problem is that this message was already out of date.
Either we inject it somewhere (think filtered resources based on the maven property version.org.wildfly) or we don't mention it. Otherwise it's unreliable (wildfly gets updated often and it's always forgotten).